lib/publishing.ts

```diff
@@ -48,11 +48,11 @@ export class PublishToMavenProject extends cdk.Construct implements IPublisher {
       platform: new LinuxPlatform(cbuild.LinuxBuildImage.UBUNTU_14_04_NODEJS_10_1_0),
       scriptDirectory: path.join(__dirname, 'publishing', 'maven'),
       entrypoint: 'publish.sh',
-      environmentVariables: {
-        STAGING_PROFILE_ID: { value: props.stagingProfileId },
-        SIGNING_KEY_SCOPE: { value: props.signingKey.scope },
-        FOR_REAL: { value: forReal },
-        MAVEN_LOGIN_SECRET: { value: props.mavenLoginSecret.secretArn }
+      env: {
```

Comment:
`env` doesn't seem to be consistent with the naming in other constructs. Maybe `environment`?